build.gradle

```diff
@@ -59,11 +58,12 @@ subprojects {
     targetCompatibility = 1.6
 
     repositories {
-        mavenCentral()
-        mavenLocal()
         maven {
-          url "https://oss.sonatype.org/content/repositories/snapshots/"
+            // The google mirror is less flaky than mavenCentral()
+            url "https://maven-central.storage-download.googleapis.com/repos/central/data/"
         }
+        mavenCentral()
```

Comment:
Why do we include `mavenCentral()` here but not in the `buildscript{}` sections? I'd expect us to be consistent, one way or the other.